fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -647,6 +647,24 @@ public void testGetChildrenWithBinary() {
     }
 
     @Test
+    public void testGetChildrenWithBinaryCustomProperties() throws RepositoryException {
+        final String pid = getRandomPid();
+        final Container container = containerService.findOrCreate(session, "/" + pid);
+        final FedoraResource resource = binaryService.findOrCreate(session, "/" + pid + "/a");
+        final javax.jcr.Node node = resource.getNode();
+        node.setProperty("dc:title", "this-is-some-title");
```

Comment:
You don't actually need the `javax.jcr.Node` but the once, so I would just do `resource.getNode().setProperty("dc:title", "this-is-some-title");`, but it's a matter of style.